vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -154,33 +148,43 @@ ent prj install --conflict-strategy=OVERRIDE
 ```
 
 ### Run a Project Locally
-1. Initialize Keycloak, which leverages docker-compose
+1. Initialize Keycloak, which leverages Docker Compose
 ``` sh
 ent prj ext-keycloak start
 ```
 2. Initialize backend microservices
 ``` sh
 ent prj be-test-run
 ```
+To fetch logs from bundle plugins, use the command
+```
+ent prj be-log
+```
+
 3. Initialize one or more frontend widgets, each from its own shell
 ``` sh
 ent prj fe-test-run
 ```
-See [this tutorial](../../tutorials/create/ms/run-local.md) for more details.
+Check out [Run Blueprint-generated Microservices and Micro Frontends in Dev Mode](../../tutorials/create/ms/run-local.md) for more details.
+
+### Custom Resources
+Entando supports a series of commands to manage custom resources. To generate a custom resource and deploy it to the current profile, use `ent ecr deploy`. The subcommand `ent ecr cr` returns the usage status of a bundle. 
+
+Refer to [Entando Custom Resources for Kubernetes](../../docs/consume/custom-resources.md) for a detailed discussion of custom resource definitions, properties and implementations.
 
 ### Get the Bundle ID and Plugin ID
-Entando will use a unique identifier for your bundle as a way to provide additional security controls around bundle-specific resources. You can determine this identifier with this command
+Entando will use a unique identifier for your bundle as a way to provide additional security controls around bundle-specific resources. You can determine this identifier with the command 
 ```sh
 ent prj get-bundle-id --auto
 ```
 
-If you have a microservice plugin in your project, a unique identifier will be calculated for it as well.
+A unique identifier will also be calculated for each microservice plugin in your project. You can determine this identifier with the command 
 ```sh
 ent prj get-plugin-id --auto --repo=<BUNDLE-REPO-URL>
 ```
-
+The helpers `get-bundle-id` and `get-plugin-id` have also been added to `ent ecr` to calculate and display the bundle and plugin names related to custom resources.
```

Comment:
Let's add this explicitly as ent ecr get-bundle-id and ent ecr get-plugin-id under the new ECR section.

Reply:
@nshaw i keep running into null errors when i enter commands like the above; do i need to specify arguments or is there missing syntax?